### DISCUSSION TITLE
Support multi-monitor systems

### DISCUSSION THREAD

**Discussion by PrOF-kk** [File-New-Project/EarTrumpet#813] [Ideas]
Currently EarTrumpet always opens on the primary monitor on multi-monitor systems. It'd be nice if it instead opened on the current monitor (either the monitor with the current active window, or the monitor where the mouse is)

**Comment by AverageComet250** [comment]
This is a good idea, however a better one, is being able to set which monitor it appears on as a config option and have an option to show it on all
